### DISCUSSION TITLE
How can I make Rootless Podman use a different directory for storage?

### DISCUSSION THREAD

**Discussion by khanetor** [containers/podman#21213] [Q&A]
I am running openSUSE with SELinux, with 2 mount points like this
<img width="859" alt="Screenshot 2024-01-09 at 21 20 03" src="https://github.com/containers/podman/assets/3459416/705bd898-9d7d-4a8e-a7da-3b26035b8b3d">

When creating Podman volume, it is using the directory `$HOME/.local/share/containers/storage/volumes`. Unfortunately, this directory is on the mount with limited capacity. This, when I was running NextCloud, I quickly ran out of disk space.

I created another directory `/var/lib/$USER/containers/storage`, and then I copied the volumes over, and tried:
1. Modifying `/etc/containers/storage.conf`, both the `rootgraph` and `rootless_storage_path`, with no success
2. Creating `$HOME/.config/containers/storage.conf`, and did the same thing without success. Volumes are still being created at the old default location.

When those still do not work, I then created a symlink from `$HOME/.local/share/containers/storage/volumes` to `/var/lib/$USER/containers/storage/volumes`. Now, volumes can be created, but containers cannot use the volume because the error is not enough space (I have already cleared out space, so this error might be SELinux related).

Could you point me to the config that I can reliably change the default storage location for rootless Podman? It is okay that I lose my data now, rather than later.

**Comment by rhatdan** [comment]
Have you tried a containers.conf file?

```
# Directory for libpod named volumes.
# By default, this will be configured relative to where containers/storage
# stores containers.
# Uncomment to change location from this default.
#
#volume_path = "/var/lib/containers/storage/volumes"
```

**Reply by schultzter** [reply]
I also had to do a `podman system reset` for this to work. And `podman system info` to make sure it got it.

**Reply by khanetor** [reply]
Yes I have, and it works. Thanks @rhatdan !
